frontend/services/src/events/eventsService.ts

```diff
@@ -136,6 +146,32 @@ export async function getAllEvents(isActive?: boolean, isPrivate?: boolean) {
   }
 }
 
+export async function getOrganiserEvents(userId: string): Promise<EventData[]> {
+  eventServiceLogger.info(`getOrganiserEvents`);
```

Comment:
we would preferably like to log this with more info e.g., logging which userId logged this etc. Use your own discretion to log the appropriate information so that it is traceable.